_data/navbars/reference.yml

```diff
@@ -49,6 +49,19 @@ section:
       path: /reference/calicoctl/node/diags
     - title: checksystem
       path: /reference/calicoctl/node/checksystem
+  - title: datastore migrate
```

Comment:
"datastore migrate" isn't one command, it's two.

`calicoctl datastore` is the subcommand.